x-pack/plugins/siem/public/store/timeline/helpers.ts

```diff
@@ -3,14 +3,16 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { getOr, omit, uniq } from 'lodash/fp';
+import { getOr, omit, uniq, isEqual, isEmpty, isEqualWith } from 'lodash/fp';
 
 import { ColumnHeader } from '../../components/timeline/body/column_headers/column_header';
 import { getColumnWidthFromType } from '../../components/timeline/body/helpers';
 import { Sort } from '../../components/timeline/body/sort';
 import {
   DataProvider,
   QueryOperator,
+  QueryMatch,
+  DataProvidersAnd,
```

Comment:
Looks like `DataProvidersAnd` isn't being used (and is causing the typecheck failure).